notebooks/growth_model_forecast.ipynb

```diff
@@ -0,0 +1,1174 @@
+{
```

Comment:
<p>Can you also add a brief <code>Conclusion</code> section to add a summary of the notebook? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>